### PR TITLE
add: to be close to assertion

### PR DESCRIPTION
**Pull request by janmeier**

### CODE REVIEW

**Review by sushantdhiman — COMMENTED** [review]
Looks Good but linting fails

File: lib/unexpected-date.js [hunk 1]
```diff
@@ -26,6 +26,32 @@ module.exports = {
...
+      }
+
+      expect.withError(function () {
+        expect(
+          subject.getTime(),
+          '[not] to be close to',
+          value.getTime(),
+          epsilon
+        );
+      }, function (e) {
```
Comment (sushantdhiman) [inline lib/unexpected-date.js]: Remove `e`

**Comment by janmeier** [comment]
Rookie mistake :D

**Comment by janmeier** [comment]
I love the fact that the assertions in the documentation are also tested - How does that even work - Built into mocha?

Also, I couldn't really find any docs for the internal unexpected API / guide to writing plugins. Did you just figure it out by trial and error?

**Comment by sushantdhiman** [comment]
Markdown is tested by using custom compiler https://github.com/sushantdhiman/unexpected-date/blob/master/test/mocha.opts , `unexpected-markdown` is used (I dont know the internals)

I took inspiration from `unexpected-moment`, also guys from Gitter channel helped

**Review by sushantdhiman — APPROVED** [review]

**Comment by sushantdhiman** [comment]
`v1.2.0`

**Pull request merged**